core/variant/callable.h

```diff
@@ -153,6 +154,7 @@ class CallableCustom {
 	virtual void call(const Variant **p_arguments, int p_argcount, Variant &r_return_value, Callable::CallError &r_call_error) const = 0;
 	virtual Error rpc(int p_peer_id, const Variant **p_arguments, int p_argcount, Callable::CallError &r_call_error) const;
 	virtual const Callable *get_base_comparator() const;
+	virtual int get_argument_count() const = 0;
```

Comment:
Keeping this virtual for now, but once I've identified the use cases I will probably make it non-virtual, just wanting to make sure I don't miss anything anywhere